recipe/migrations/cuda130.yaml

```diff
@@ -0,0 +1,62 @@
+migrator_ts: 1755016036
+__migrator:
+  operation: key_add
+  migration_number:
+    1
+  build_number:
+    1
+  paused: true
+  override_cbc_keys:
+    - cuda_compiler_stub
+  check_solvable: false
+  primary_key: cuda_compiler_version
+  ordering:
+    cuda_compiler_version:
+      - 12.4
+      - 12.6
+      - 12.8
+      - None
+      - 12.9
+      - 13.0
+      # to allow manual opt-in for CUDA 11.8, see
+      # https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/7472
+      # must be last due to how cuda_compiler ordering in that migrator works
+      - 11.8
+  wait_for_migrators:
+    - cuda129
+  commit_message: |
+    Upgrade to CUDA 13.0
+    
+    CUDA 13.0 requires architecture `sm_75` or higher, and renamed `sm_101` to
+    `sm_110`. To build for these, maintainers will need to modify their existing list of
+    specified architectures (e.g. `CMAKE_CUDA_ARCHITECTURES`, `TORCH_CUDA_ARCH_LIST`, etc.)
+    for their package.
+
```

Comment:
Insert note here about binary path changes on Windows

Reply:
Do you want to do this before merging the still-paused migration? We could also add that text later.

Reply:
No, I don't want to add the note because we may be reverting the path changes on Windows.